modular_citadel/code/modules/mob/cit_emotes.dm

```diff
@@ -87,7 +86,7 @@
 	emote_type = EMOTE_AUDIBLE
 
 /datum/emote/living/snap/run_emote(mob/living/user, params)
-	if(ishuman(user))
+	if(iscarbon(user) || ispAI(user))
```

Comment:
Shitty citadel emotes.
Move the parent call on top, then add an early return if **.** is FALSE.
Then remove this typecheck.